lighthouserc.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+    ci: {
+        upload: {
+            target: 'temporary-public-storage'
+        },
+        collect: {
+            staticDistDir: './public', // location of static files for lighthouse CI to serve
```

Comment:
not sure we actually have a static site in the way it expects us to?

Reply:
True dat. isSinglePageApplication: true